versioned_docs/version-3.13/whats-new.md

```diff
@@ -0,0 +1,81 @@
+---
+title: What's New in RabbitMQ 3.13
+---
+<!--
+Copyright (c) 2024 Broadcom. All Rights Reserved. The term "Broadcom" refers
+to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+import {
+  RabbitMQServerVersion,
+} from '@site/src/components/RabbitMQServer';
+
+import MQTTLogo from '@site/static/img/mqtt-logo/mqtt-ver.svg';
+import KhepriLogo from '@site/static/img/khepri-logo.svg';
+
+# What's New in RabbitMQ 3.13
+
+RabbitMQ 3.13 is a new release branch [published on 22 Feb
+2024](/blog/2024/03/11/rabbitmq-3.13.0-announcement). <strong>RabbitMQ
+<RabbitMQServerVersion/></strong> is the latest patch release for this release
+branch. See the [Release Information section](/release-information) to learn
+more about all patch releases for this release branch.
+
+## MQTT 5.0 support
+
+<MQTTLogo className="floating-logo"/>
+
+MQTT is the standard protocol for the Internet of Things (IoT).
+
+IoT remote devices can have a poor network quality when connecting to the
+broker. Therefore, MQTT is lightweight: MQTT protocol headers are small to save
+network bandwidth. Since IoT devices might often disconnect and reconnect -
+imagine a car driving through a tunnel - MQTT is also efficient: Clients are
+connected and authenticated with a handshake that is shorter compared to other
+messaging protocols.
+
+[MQTT 5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html) is
+the latest version of the protocol and [is now supported by
+RabbitMQ 3.13.0](/blog/2023/07/21/mqtt5)!
+
+## Stream filtering
+
+[Stream filtering](/blog/2023/10/16/stream-filtering) allows to save bandwidth
+between the broker and consuming applications when those applications need only
+a subset of the messages of a stream.
+
+## Classic Queue Performance Improvements
+
+Classic queues can use [the newer message store
+v2](/blog/2024/01/11/3.13-release) which brings significant performance
+improvements, in terms of publishing/consuming rates, latency and memory
+footprint.
+
```

Comment:
ok when I looked at this on preview: https://2667a207.rabbitmq-website.pages.dev/docs/whats-new I didn't know what this new message store was? the name of it etc, I had to go the blog to find out more, I then simply searched for "Message Store" .

Maybe under the title make the text that you have under this heading the 1st bullet point and then have a 2nd bullet point for "classic queue storage format version 1 is deprecated" and provide a one line how this results in classic queue performance improvements

Reply:
So when I read this: "Classic queues can use [the newer message store v2](https://3564b8ce.rabbitmq-website.pages.dev/blog/2024/01/11/3.13-release)", the first thing I do is click the link, I am then looking for a heading: "Message Store" or something like it to take me directly to more information this "newer v2 message store", I know the section is this one: https://3564b8ce.rabbitmq-website.pages.dev/blog/2024/01/11/3.13-release#classic-queues-storage-primer but this section in the blog is extremely long...multiple paragraphs and the punch line of what I am looking for is here: 

The per-vhost message store is still there for larger messages, but version 3.13 changes its behaviour significantly.

_For backwards compatibility queue_index_embed_msgs_below still controls whether a message is large enough to be stored in the per-vhost message store and the default is still 4kB.

It's worth mentioning that the per-vhost message store greatly improves support for fanout scenarios (when a single message is delivered to multiple classic queues), since RabbitMQ can just write the message once, rather than once per queue._

I guess what I am saying is the snippet you have in the "What's New" is ok but you have to go "the detail" in the blog to find out more about this "newer message store", the blog information needs to be clearer to get the user to find this information faster. 

Can you update the blog to do this? So define exactly what this "newer message store V2" is at the top of the blog and describe its benefits, then next, we can go on to explain briefly how classic queues store messages (or do we even need to do that), if we do perhaps we can shorten this section more.

Reply:
or else @dumbbell do this in the "Classic Queue Performance Improvements" section in the "What's new", define exactly what this "newer message store V2" is at the top of the blog and describe its benefits and then link out to the blog. For future blogs we should use a more structured approach to the sections i.e. Title = feature name -> short opening paragraph = a couple of lines explaining clearly what this new feature is -> provide some benefits -> provide an example, then we can pull this information (well without the example) directly from the blog and into our What's New.

Reply:
I see. I don't know the change well enough to explain clearly what it is and the benefits and don't want give an incorrect summary. @essen, could you please write a short paragraph that explains what the message store v2 is and its benefits?

Reply:
This refers/links to this blog post: https://www.rabbitmq.com/blog/2024/01/11/3.13-release

Isn't that good enough? :)

It shouldn't be "can use" but "use" though. The 3.13 changes are for all CQs. It also shouldn't say "v2" as the 3.13 changes don't fundamentally change the message store. So something like

```
Classic queues [shared message store](/blog/2024/01/11/3.13-release) brings significant performance
improvements, in terms of ...

Reply:
Thanks @lhoguin, I applied your suggestion!

@pstack2021: What do you think?